examples/nextjs/src/createEmotionCache.js

```diff
@@ -1,5 +1,8 @@
 import createCache from '@emotion/cache';
 
 export default function createEmotionCache() {
+  // If using css-modules, to correct the injection order
+  // add prepend: true to the createCache options
+  // https://mui.com/guides/interoperability/#css-injection-order-3
```

Comment:
AFAIK, It's not enough, people also need to update the injection order in:

https://github.com/mui-org/material-ui/blob/a04f8a0b17acce24fa466c43a7ac8d6c01bad7a3/examples/nextjs/pages/_document.js#L84

Reply:
I dig a little bit deeper. At the built time, emotion styles are not at the top of `head` even though `prepend: true` is set (this is due to the styles provided to nextjs). When the site is hydrated on the client, emotion moves styles up to head if `prepend: true` is set. 

ref: https://github.com/emotion-js/emotion/blob/01cca604ca93c5fd3958c41045ecbcac72b65480/packages/sheet/src/index.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L97